src/com/google/javascript/jscomp/RewriteJsonToModule.java

```diff
@@ -36,6 +36,8 @@
     implements CompilerPass {
   public static final DiagnosticType JSON_UNEXPECTED_TOKEN =
       DiagnosticType.error("JSC_JSON_UNEXPECTED_TOKEN", "Unexpected JSON token");
+  public static final String JSC_BROWSER_BLACKLISTED_MARKER = "$jscomp$browser$blacklisted";
```

Comment:
just did this in a separate commit for easier review. Let me know if it looks good and I'll squash them